components/channel_header/channel_header.tsx

```diff
@@ -539,7 +637,7 @@ class ChannelHeader extends React.PureComponent {
                                 >
                                     {(message) => (
                                         <i
-                                            aria-label={message}
+                                            aria-label={message as unknown as string}
```

Comment:
in this case it would be better to use the `formatMessage` function from `react-intl` and pass it into the icon